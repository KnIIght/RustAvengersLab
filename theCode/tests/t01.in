1
Steve
enhanced strength and durability
my vibranium shield
