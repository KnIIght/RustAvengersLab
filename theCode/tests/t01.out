My name is Steve, my power is enhanced strength and durability, my weapon is my vibranium shield.
I could do this all day.
Steve is already worthy...
